### PR TITLE
7394 depth chart render

### PR DESCRIPTION
**Pull request by stephensprinkle**
[Clubhouse Story](https://app.clubhouse.io/augur/story/7394)

## Submitter checklist
- [x] Test covering functionality added/fixed in
- Check the linked story includes the following:
  - [x] Steps to reproduce
  - [x] Screenshot (If applicable)

## Reviewer Checklist
- [x] Test/lint pass 
- [ ] Post merge, story marked complete

### CODE REVIEW

**Comment by stephensprinkle** [comment]
I couldn’t repro that deep link issue — could you paste the error logging?

File: src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx [hunk 1]
```diff
@@ -281,3 +267,23 @@ export default class MarketOutcomeDepth extends Component {
     )
   }
 }
+
+function nearestCompletelyFillingOrder(price, marketDepth) {
```
[the remark this answers is not in the document]
Comment (stephensprinkle) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Fixed w/ 59b3cdc85ec312c037f1ae0266b441f97be5f639

**Comment by justinbarry** [comment]
```
market-trading--form.jsx?8b3c:200 Uncaught TypeError: Cannot read property 'name' of undefined
    at MarketTradingForm.render (market-trading--form.jsx?8b3c:200)
    at finishClassComponent (react-dom.development.js?cada:7873)
    at updateClassComponent (react-dom.development.js?cada:7850)
    at beginWork (react-dom.development.js?cada:8225)
    at performUnitOfWork (react-dom.development.js?cada:10224)
    at workLoop (react-dom.development.js?cada:10288)
    at HTMLUnknownElement.callCallback (react-dom.development.js?cada:542)
    at Object.invokeGuardedCallbackDev (react-dom.development.js?cada:581)
    at invokeGuardedCallback (react-dom.development.js?cada:438)
    at renderRoot (react-dom.development.js?cada:10366)
```

File: src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (justinbarry) [reply src/modules/market/components/market-outcome-charts--depth/market-outcome-charts--depth.jsx]: Looks good.

**Comment by justinbarry** [comment]
Cannot reproduce any longer.

**Pull request merged**